meta.json

```diff
@@ -26,9 +26,15 @@
        {
         "api": "rdk:component:sensor",
         "model": "viam:lorawan:dragino-LHT65N",
-        "markdown_link": "README.md#configure-your-viamlorawandragino-lht65n",
+        "markdown_link": "README.md#configure-your-dragino-sensor",
         "short_description": "A sensor component for a dragino LHT65N LoRaWAN node."
      },
+     {
+      "api": "rdk:component:sensor",
+      "model": "viam:lorawan:dragino-WQS-LB",
+      "markdown_link": "README.md#configure-your-dragino-sensor",
+      "short_description": "A sensor component for a dragino WQS-LB LoRaWAN node."
```

Comment:
@JohnN193 where in the meta.json do we add the DoCommands from the readme?

Reply:
It’s part of the markdown link. The module generator includes DoCommands in the readme.

Reply:
This module already displays most DoCommands via the markdown link. This pr was just not including them yet

Reply:
Thanks for explaining!